src/components/report/ReportHeader.svelte

```diff
@@ -43,7 +44,7 @@ Based on {catalog.title}
     {#if $evaluation["author"]["website"]}<li>Website: <a href="{$evaluation['author']['website']}" target="_blank">{$evaluation["author"]["website"]} <span class="visuallyhidden">(opens in a new window or tab)</span></a></li>{/if}
   </ul>
 {/if}
-{#if $evaluation["vendor"]}
+<!-- {#if $evaluation["vendor"]}
```

Comment:
We remove this because we're branding it, so it is clear?

Reply:
Ya in #13 I provided a surrounding div to allow this to be hidden with CSS if needed.